manager/director/apps/sites/forms.py

```diff
@@ -143,10 +143,17 @@ class DomainForm(forms.Form):
         required=False,
         validators=[
             validators.RegexValidator(
-                regex=r"^(?!(.*\.)?sites\.tjhsst\.edu$)[0-9a-zA-Z_\- .]+$",
+                regex=r"^.*sites\.tjhsst\.edu$",
```

Comment:
The previous regex specifically did `(.*\.)?sites\.tjhsst\.edu$` to allow things like `test-sites.tjhsst.edu` while blocking `sites.tjhsst.edu` and `a.sites.tjhsst.edu`. I'd recommend keeping that.

Nice work with `inverse_match`, though. I didn't know about that.